File: source/js/models/project-task.js

var ProjectTask = function (projectTaskData) {
    var self = this;

    self.name = '';
    self.completed = false;
    self.hours = 0;

    if (projectTaskData) {
        self.name = projectTaskData.name;
        self.completed = projectTaskData.completed;
        self.hours = projectTaskData.hours;
    }
};

File: ef-overview/Driver/Data/Models/ProjectTask.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace Data.Models
{
    public class ProjectTask
    {
        public int Id { get; set; }
        public int ProjectId { get; set; }
        public string Name { get; set; }
        public decimal? Hours { get; set; }
        public bool Completed { get; set; }

        public Project Project { get; set; }
    }
}
